docs/usage/string-pattern-matching.md

```diff
@@ -5,6 +5,38 @@ Renovate string matching syntax for some configuration options allows you, as us
 - [`minimatch`](https://github.com/isaacs/minimatch) glob patterns, including exact strings matches
 - regular expression (regex) patterns
 
+The following fields support this pattern matching:
+
+- `allowedEnv`
```

Comment:
We should make this a field in options.ts when we have migrated. This would make the validation also more reliable and prevents us from maintaining this manually.